### PR TITLE
Issue/27 landing page

### PR DESCRIPTION
**Pull request by Florian-Sabonchi**
First draft for landing Page with Contact Page

### CODE REVIEW

File: landing-page/styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+}
+
+.text-secondary{
+  color: #5D4D80;
+}
+
+
+.bg-team{
+  background: #DDDDDD;
+}
```
Comment (JannikWibker) [inline landing-page/styles/globals.css:20]: use the tailwind config instead: https://tailwindcss.com/docs/theme

File: landing-page/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <nav className="bg-primary relative flex flex-wrap items-center justify-between px-2 py-3 mb-3">
+        <a href="/" className="flex items-center">
+          <img src="https://raw.githubusercontent.com/Just-another-Muensterhack/helpwave-static/main/banner.svg" className="h-6 mr-3 sm:h-9" alt="Flowbite Logo"/>
+        </a>
+        <button data-collapse-toggle="navbar-default" type="button"
+                className="inline-flex items-center p-2 ml-3 text-sm text-white rounded-lg md:hidden  focus:outline-none focus:ring-2 focus:ring-transparent dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600"
+                aria-controls="navbar-default" aria-expanded="false">
+          <span className="sr-only">Open main menu</span>
+          <svg className="w-6 h-6" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20"
+               xmlns="http://www.w3.org/2000/svg">
```
Comment (JannikWibker) [inline landing-page/components/Navbar.tsx:16]: xmlns attributes can be removed

File: landing-page/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                   Jonas
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Project Management
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
```
Comment (JannikWibker) [inline landing-page/components/Footer.tsx:151]: this is **really** repetitive, use a custom component and use it to map over an array

File: landing-page/components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    <footer className="p-4 bg-white shadow md:px-6 md:py-8 dark:bg-gray-900">
+      <div className="sm:flex">
+        <span className="underline">Our Partners</span>
+        <img src="https://muensterhack.de/img/logo_muensterhack.png"
+             className="mt-5 sm:mt-0 ml-5 h-8" alt="muensterhack Logo"/>
+        <img src="https://www.ukm.de/typo3conf/ext/ukm_main/Resources/Public/img/UKM_LOGO_2.jpg"
+             className="mt-5 sm:mt-0  ml-5 h-8" alt="ukm Logo"/>
+        <img src="https://www.warendorf.de/fileadmin/templates/gfx/logo.svg"
+             className="mt-5 sm:mt-0  ml-5 h-8" alt="warendorf Logo" />
+      </div>
```
Comment (JannikWibker) [inline landing-page/components/Footer.tsx:12]: these logos shouldn't be loaded cross domains

File: landing-page/components/ContactForm.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <section className="bg-white">
+      <div className="py-8 lg:py-16 px-4 mx-auto max-w-screen-md">
+        <h2 className="mb-4 text-4xl tracking-tight font-extrabold text-center text-gray-900 dark:text-white">
+          Contact Us</h2>
+        <p className="mb-8 lg:mb-16 font-light text-center text-gray-500 dark:text-gray-400 sm:text-xl">Got a technical
+          issue? Want to send feedback about a beta feature? Need details about our Business plan? Let us know.</p>
+        <form action="#" className="space-y-8">
+          <div>
+            <label htmlFor="email" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your
+              email</label>
```
Comment (JannikWibker) [inline landing-page/components/ContactForm.tsx:13]: don't know if we want to deal with a lot of different screen sizes at the moment; maybe better to not have any kind of media-queries in here at this time

File: landing-page/components/Carousel.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+export const Carousel = () => {
+  return (
+    <div id="carouselExampleCrossfade" className="carousel slide carousel-fade relative" data-bs-ride="carousel">
+      <div className="carousel-inner relative w-full overflow-hidden">
+        <div className="carousel-item  float-left w-full">
+          <div className="flex h-96" data-carousel-item="active">
```
Comment (JannikWibker) [inline landing-page/components/Carousel.tsx:6]: `id="carouselExampleCrossfade"` as well as some of the classnames aren't used for anything -> remove

(this applies to other files as well)

File: landing-page/pages/contact.tsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+const Home: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>Contact</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Navbar></Navbar>
+      <ContactForm></ContactForm>
```
Comment (JannikWibker) [inline landing-page/pages/contact.tsx:15]: use `<Navbar />` instead of `<Navbar></Navbar>`, same for `ContactForm`

**Review by JannikWibker — CHANGES_REQUESTED** [review]
`Module not found: Can't resolve '../components/Body'`

**Comment by JannikWibker** [comment]
PRs doing any UI work should always include a screenshot

**Comment by Florian-Sabonchi** [comment]
image

**Comment by Florian-Sabonchi** [comment]
image

**Comment by use-to** [comment]
@Florian-Sabonchi I really like the affort you're putting into this - keep it up!
<details>
  <summary>This is the cloudflare pages log</summary>
```
12:22:42.076 | Cloning repository...
-- | --
12:22:43.008 | From https://github.com/helpwave/web
12:22:43.008 | * branch            03a2dfb44e6b2ff6e998469b7c8c9183dc725a01 -> FETCH_HEAD
12:22:43.009 |  
12:22:43.075 | HEAD is now at 03a2dfb feat: code review
12:22:43.076 |  
12:22:43.227 |  
12:22:43.256 | Success: Finished cloning repository files
12:22:43.927 | Installing dependencies
12:22:43.939 | Python version set to 2.7
12:22:47.436 | v12.18.0 is already installed.
12:22:48.675 | Now using node v12.18.0 (npm v6.14.4)
12:22:48.902 | Started restoring cached build plugins
12:22:48.918 | Finished restoring cached build plugins
12:22:49.418 | Attempting ruby version 2.7.1, read from environment
12:22:53.044 | Using ruby version 2.7.1
12:22:53.418 | Using PHP version 5.6
12:22:53.582 | 5.2 is already installed.
12:22:53.610 | Using Swift version 5.2
12:22:53.610 | Started restoring cached node modules
12:22:53.625 | Finished restoring cached node modules
12:22:53.876 | Installing NPM modules using NPM version 6.14.4
12:23:12.994 |  
12:23:12.995 | > core-js-pure@3.26.1 postinstall /opt/buildhome/repo/node_modules/core-js-pure
12:23:12.995 | > node -e "try{require('./postinstall')}catch(e){}"
12:23:12.995 |  
12:23:13.111 | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
12:23:13.111 |  
12:23:13.111 | The project needs your help! Please consider supporting of core-js:
12:23:13.111 | > https://opencollective.com/core-js
12:23:13.112 | > https://patreon.com/zloirock
12:23:13.112 | > bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz
12:23:13.112 |  
12:23:13.112 | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
12:23:13.112 |  
12:23:13.357 | npm notice created a lockfile as package-lock.json. You should commit this file.
12:23:13.359 | npm WARN notsup Unsupported engine for next@12.3.1: wanted: {"node":">=12.22.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.360 | npm WARN notsup Not compatible with your version of node/npm: next@12.3.1
12:23:13.360 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-android-arm64@12.3.1 (node_modules/next/node_modules/@next/swc-android-arm64):
12:23:13.361 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-android-arm64@12.3.1: wanted {"os":"android","arch":"arm64"} (current: {"os":"linux","arch":"x64"})
12:23:13.361 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-android-arm-eabi@12.3.1 (node_modules/next/node_modules/@next/swc-android-arm-eabi):
12:23:13.361 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-android-arm-eabi@12.3.1: wanted {"os":"android","arch":"arm"} (current: {"os":"linux","arch":"x64"})
12:23:13.361 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-darwin-arm64@12.3.1 (node_modules/next/node_modules/@next/swc-darwin-arm64):
12:23:13.361 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-darwin-arm64@12.3.1: wanted {"os":"darwin","arch":"arm64"} (current: {"os":"linux","arch":"x64"})
12:23:13.362 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-darwin-x64@12.3.1 (node_modules/next/node_modules/@next/swc-darwin-x64):
12:23:13.362 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-darwin-x64@12.3.1: wanted {"os":"darwin","arch":"x64"} (current: {"os":"linux","arch":"x64"})
12:23:13.362 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-freebsd-x64@12.3.1 (node_modules/next/node_modules/@next/swc-freebsd-x64):
12:23:13.362 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-freebsd-x64@12.3.1: wanted {"os":"freebsd","arch":"x64"} (current: {"os":"linux","arch":"x64"})
12:23:13.362 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-linux-arm64-musl@12.3.1 (node_modules/next/node_modules/@next/swc-linux-arm64-musl):
12:23:13.363 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-linux-arm64-musl@12.3.1: wanted {"os":"linux","arch":"arm64"} (current: {"os":"linux","arch":"x64"})
12:23:13.363 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-linux-arm-gnueabihf@12.3.1 (node_modules/next/node_modules/@next/swc-linux-arm-gnueabihf):
12:23:13.363 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-linux-arm-gnueabihf@12.3.1: wanted {"os":"linux","arch":"arm"} (current: {"os":"linux","arch":"x64"})
12:23:13.363 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-win32-arm64-msvc@12.3.1 (node_modules/next/node_modules/@next/swc-win32-arm64-msvc):
12:23:13.363 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-win32-arm64-msvc@12.3.1: wanted {"os":"win32","arch":"arm64"} (current: {"os":"linux","arch":"x64"})
12:23:13.364 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-linux-arm64-gnu@12.3.1 (node_modules/next/node_modules/@next/swc-linux-arm64-gnu):
12:23:13.364 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-linux-arm64-gnu@12.3.1: wanted {"os":"linux","arch":"arm64"} (current: {"os":"linux","arch":"x64"})
12:23:13.364 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-win32-ia32-msvc@12.3.1 (node_modules/next/node_modules/@next/swc-win32-ia32-msvc):
12:23:13.364 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-win32-ia32-msvc@12.3.1: wanted {"os":"win32","arch":"ia32"} (current: {"os":"linux","arch":"x64"})
12:23:13.364 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: @next/swc-win32-x64-msvc@12.3.1 (node_modules/next/node_modules/@next/swc-win32-x64-msvc):
12:23:13.364 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @next/swc-win32-x64-msvc@12.3.1: wanted {"os":"win32","arch":"x64"} (current: {"os":"linux","arch":"x64"})
12:23:13.365 | npm WARN notsup Unsupported engine for @typescript-eslint/eslint-plugin@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.365 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/eslint-plugin@5.46.0
12:23:13.365 | npm WARN notsup Unsupported engine for @typescript-eslint/parser@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.365 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/parser@5.46.0
12:23:13.365 | npm WARN notsup Unsupported engine for eslint@8.24.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.365 | npm WARN notsup Not compatible with your version of node/npm: eslint@8.24.0
12:23:13.365 | npm WARN notsup Unsupported engine for eslint-plugin-promise@6.1.1: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.365 | npm WARN notsup Not compatible with your version of node/npm: eslint-plugin-promise@6.1.1
12:23:13.366 | npm WARN notsup Unsupported engine for eslint-plugin-n@15.6.0: wanted: {"node":">=12.22.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.366 | npm WARN notsup Not compatible with your version of node/npm: eslint-plugin-n@15.6.0
12:23:13.367 | npm WARN notsup Unsupported engine for @typescript-eslint/scope-manager@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.368 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/scope-manager@5.46.0
12:23:13.368 | npm WARN notsup Unsupported engine for @typescript-eslint/utils@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.368 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/utils@5.46.0
12:23:13.368 | npm WARN notsup Unsupported engine for @typescript-eslint/type-utils@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.368 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/type-utils@5.46.0
12:23:13.369 | npm WARN notsup Unsupported engine for @typescript-eslint/types@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.369 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/types@5.46.0
12:23:13.369 | npm WARN notsup Unsupported engine for @typescript-eslint/visitor-keys@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.369 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/visitor-keys@5.46.0
12:23:13.369 | npm WARN notsup Unsupported engine for eslint-visitor-keys@3.3.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.369 | npm WARN notsup Not compatible with your version of node/npm: eslint-visitor-keys@3.3.0
12:23:13.369 | npm WARN notsup Unsupported engine for @typescript-eslint/typescript-estree@5.46.0: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.369 | npm WARN notsup Not compatible with your version of node/npm: @typescript-eslint/typescript-estree@5.46.0
12:23:13.370 | npm WARN notsup Unsupported engine for @eslint/eslintrc@1.3.3: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.370 | npm WARN notsup Not compatible with your version of node/npm: @eslint/eslintrc@1.3.3
12:23:13.370 | npm WARN notsup Unsupported engine for @humanwhocodes/module-importer@1.0.1: wanted: {"node":">=12.22"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.370 | npm WARN notsup Not compatible with your version of node/npm: @humanwhocodes/module-importer@1.0.1
12:23:13.370 | npm WARN notsup Unsupported engine for eslint-scope@7.1.1: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.370 | npm WARN notsup Not compatible with your version of node/npm: eslint-scope@7.1.1
12:23:13.370 | npm WARN notsup Unsupported engine for espree@9.4.1: wanted: {"node":"^12.22.0 \|\| ^14.17.0 \|\| >=16.0.0"} (current: {"node":"12.18.0","npm":"6.14.4"})
12:23:13.370 | npm WARN notsup Not compatible with your version of node/npm: espree@9.4.1
12:23:13.370 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.2 (node_modules/chokidar/node_modules/fsevents):
12:23:13.371 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
12:23:13.371 |  
12:23:13.371 | added 337 packages from 291 contributors and audited 353 packages in 18.542s
12:23:13.481 |  
12:23:13.482 | 101 packages are looking for funding
12:23:13.482 | run `npm fund` for details
12:23:13.482 |  
12:23:13.482 | found 0 vulnerabilities
12:23:13.482 |  
12:23:13.505 | NPM modules installed
12:23:13.906 | Installing Hugo 0.54.0
12:23:15.956 | Hugo Static Site Generator v0.54.0-B1A82C61A/extended linux/amd64 BuildDate: 2019-02-01T10:04:38Z
12:23:15.960 | Started restoring cached go cache
12:23:15.985 | Finished restoring cached go cache
12:23:16.152 | go version go1.14.4 linux/amd64
12:23:16.170 | go version go1.14.4 linux/amd64
12:23:16.173 | Installing missing commands
12:23:16.174 | Verify run directory
12:23:16.174 | Executing user command: next build && next export
12:23:16.746 | (node:1345) ExperimentalWarning: The ESM module loader is experimental.
12:23:16.752 | warn  - No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
12:23:16.797 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
12:23:16.797 | This information is used to shape Next.js' roadmap and prioritize features.
12:23:16.797 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
12:23:16.798 | https://nextjs.org/telemetry
12:23:16.798 |  
12:23:16.935 | info  - Linting and checking validity of types...
12:23:19.435 | warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config
12:23:19.991 |  
12:23:19.991 | Failed to compile.
12:23:19.993 |  
12:23:20.004 | ./components/Carousel.tsx
12:23:20.007 | 11:34  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
12:23:20.007 |  
12:23:20.007 | info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
12:23:20.019 | Failed: build command exited with code: 1
12:23:20.877 | Failed: an internal error occurred
```
</details>

**Comment by use-to** [comment]
> 

@JannikWibker already mentioned that this error may occur frequently.

**Review by use-to — APPROVED** [review]

**Review by JannikWibker — APPROVED** [review]

**Pull request merged**